src/style.css

```diff
@@ -1,3 +1,86 @@
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: #ebebe4;
+  font-family: Roboto, Inter, Helvetica, sans-serif;
+  box-sizing: border-box;
```

Comment:
Usually we apply border box property only once for all elements on a page.
```
*, *::before, *::after {
  box-sizing: border-box;
}
```
See details here https://css-tricks.com/almanac/properties/b/box-sizing/